sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -235,11 +241,54 @@ def run_pipeline(self, pipeline):
     # are known to be KVs.
     from apache_beam.runners.dataflow.dataflow_runner import DataflowRunner
     pipeline.visit(DataflowRunner.group_by_key_input_visitor())
+    self._bundle_repeat = self._bundle_repeat or pipeline._options.view_as(
```

Comment:
Do runners have another way to access options? Do they need to use pipeline._options?